### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by hamedrnik** [code-tree/imitation#1]
(gedit:9313): Gtk-WARNING **: Invalid text buffer iterator: either the iterator is uninitialized, or the characters/pixbufs/widgets in the buffer have been modified since the iterator was created.
You must use marks, character numbers, or line numbers to preserve a position across buffer modifications.
You can apply tags and insert marks without invalidating your iterators,
but any mutation that affects 'indexable' buffer contents (contents that can be referred to by character offset)
will invalidate all outstanding iterators

**Comment by code-tree** [comment]
Thanks for the report. Did you by chance have any find-and-replace highlighting present at the time? (as it usually happens for me during that).

As recently added to the README, I'm afraid I don't have the time to maintain this plugin any more, but if you find a solution please make a pull request.

**Comment by hamedrnik** [comment]
@code-tree Yes It happens during find-and-replace. I found something but I'm sure it's related to this issue or not (http://stackoverflow.com/a/4930118).
If I get a chance I will look into it during next weekend. BTW, thanks for this cool plug-in.
